src/replace/imitate.coffee

```diff
@@ -16,3 +16,6 @@ module.exports = (realThing, optionalName) ->
     tdFunction(if realThing?.name then realThing.name else optionalName)
   else
     log.error("td.replace", "\"#{optionalName}\" property was found, but test double only knows how to replace functions, constructors, & objects containing functions (its value was #{stringifyAnything(realThing)}).")
+
+module.exports.canImitate = (realThing) -> 
```

Comment:
The other option here is to accept an arg that would skip erroring on demand, but logic would still be required to avoid adding empty imitations to the fakeThing.